### PR TITLE
Added question to FAQ on delays discrepancy

### PR DESCRIPTION
**Pull request by rdowinton**
Added question to FAQ on how request speed differs when using Crawlera as opposed to routing requests through proxies directly.

@qrilka can you please review? Thanks

### CODE REVIEW

File: crawlera.rst [hunk 1]
```diff
@@ -453,3 +453,10 @@ Where can I monitor my Crawlera usage?
 --------------------------------------
 
 Go to your Scrapinghub home page (i.e. ``https://dash.scrapinghub.com/USERNAME/``), and you should see your Crawlera accounts in the Crawlera section. If you click on a user, you will be able review the number of requests per day/month for that user.
+
+Why are requests slower through Crawlera as opposed to using proxies directly?
+------------------------------------------------------------------------------
+
+If you're using your own proxies, you may notice a discrepancy in speed between using your own proxies and using them with Crawlera. This is because there is some overhead when routing requests through Crawlera as the request has to go through the Crawlera endpoint before going through the proxy.
+
+Crawlera also throttles requests and introduces delays to avoid being banned on the target website. These delays can differ depending on the target domain, as some popular sites have more rigorous anti-scraping measures than others.
```
Comment (qrilka) [inline crawlera.rst]: It's not "throttles and introduces delays" but rather "it throttles by introducing delays".
Also it's not only about antiban measures - e.g. rate limit could serve as a way against bringing down weak webservers.

**Comment by qrilka** [comment]
@rdowinton also we need to note that retries could add another reason for extra delays in Crawlera.

File: crawlera.rst [hunk 2]
```diff
@@ -453,3 +453,10 @@ Where can I monitor my Crawlera usage?
 --------------------------------------
 
 Go to your Scrapinghub home page (i.e. ``https://dash.scrapinghub.com/USERNAME/``), and you should see your Crawlera accounts in the Crawlera section. If you click on a user, you will be able review the number of requests per day/month for that user.
+
+Why are requests slower through Crawlera as opposed to using proxies directly?
+------------------------------------------------------------------------------
+
+If you're using your own proxies, you may notice a discrepancy in speed between using your own proxies and using them with Crawlera. This is because there is some overhead when routing requests through Crawlera as the request has to go through the Crawlera endpoint before going through the proxy.
```
Comment (pablohoffman) [inline crawlera.rst]: I would not mention the crawlera master overhead, or at least mention it less prominently than the thortlling (now it's mentioned first).

**Comment by rdowinton** [comment]
Updated PR. Removed mention of Crawlera master overhead and described how throttling will also help prevent inadvertently taking down the target website.

**Pull request merged**